client/src/components/Dashboard/Products/ProductCard.js

```diff
@@ -17,6 +17,7 @@ export default function ProductCard(props) {
   const user = useSelector((state) => state.user.user);
   const goToProduct = (item) => {
     const productURL = RoutePaths.Product.replace(":name", item.name);
+    console.log("from product card", item);
```

Comment:
If this console log was used for debugging, it's probably best to remove it now

Reply:
removed